ddl/index.go

```diff
@@ -755,7 +755,11 @@ func IngestJobsNotExisted(ctx sessionctx.Context) bool {
 // tryFallbackToTxnMerge changes the reorg type to txn-merge if the lightning backfill meets something wrong.
 func tryFallbackToTxnMerge(job *model.Job, err error) {
 	if job.State != model.JobStateRollingback {
-		logutil.BgLogger().Info("[ddl] fallback to txn-merge backfill process", zap.Error(err))
+		if err != nil {
+			logutil.BgLogger().Warn("[ddl] fallback to txn-merge backfill process", zap.Error(err))
+		} else {
+			logutil.BgLogger().Info("[ddl] fallback to txn-merge backfill process")
+		}
```

Comment:
Sorry, I just found that this is the dead code.. Please remove this : )

Reply:
Done